src/gql/queries/index.ts

```diff
@@ -0,0 +1,18 @@
+export { GET_USER_CONFIG } from "./get-user-config";
```

Comment:
What's the advantage of having this over importing the queries from where they're defined?

Reply:
Its a popular pattern. It just cleans up imports instead of having to import multiple files from similar paths, you instead have one origin for them. We already do the same thing in the hooks folder. https://github.com/evergreen-ci/spruce/blob/new-ui-feature-branch/src/hooks/index.ts and in some other component folders.